tutorials/drybell/nlp_lf.py

```diff
@@ -39,17 +39,11 @@
 ]
 
 
-class CombineTextPreprocessor(Preprocessor):
-    def __init__(self, title_field, body_field, article_field):
-        super().__init__(
-            dict(title=title_field, body=body_field), dict(article=article_field)
-        )
+@preprocessor(dict(title="title", body="body"), dict(article="article"))
+def combine_text_preprocessor(title: str, body: str) -> FieldMap:
+    return dict(article=f"{title} {body}")
```

Comment:
Okay here's what's bugging me: this decorator is still super clunky.  Why not [get the input fields from the preprocess args](https://stackoverflow.com/questions/218616/getting-method-parameter-names-in-python) and just let the user implicitly define the output fields by the return statement?  Then you'd have a super nice interface where you only would need to add a `@preprocessor` decorator here!

Currently, there's just so much redundant info being written down for a simple op...

Reply:
Good call, I'll implement this for LambdaPreprocessor. Might take more thought for Preprocessor, and out-of-scope for this diff.